Bio/PDB/__init__.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2002, Thomas Hamelryck (thamelry@binf.ku.dk)
+# Copyright (C) 2002,2020 Thomas Hamelryck (thamelry@binf.ku.dk), Simon Duerr
```

Comment:
I would leave the copyright notice as is. We've had many contributions and it'd be a very long line if we included everyone's name. That's why we have the `CONTRIBUTORS.md` file and also why I wrote the list of "major" contributors below (in alphabetical order).

Reply:
And when we do have multiple authors listed, we typically have one name per line. e.g.

```
# Copyright (C) 2002, Thomas Hamelryck (thamelry@binf.ku.dk)
# Copyright (C) 20201, Simon Duerr
```

This scales better, and makes things like merge conflicts less likely and easier to resolve.